onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -47,27 +47,66 @@ def _skip_layer_norm_pattern(op, input, skip, gamma, beta, epsilon, stash_type):
         epsilon=epsilon,
         stash_type=stash_type,
     )
-    return normalized
+    return normalized, skip_sum
 
 
 def _skip_layer_normalization(op, input, skip, gamma, beta, epsilon, stash_type):
     if stash_type.value != 1:  # FLOAT type
         return None
-    normalized, _mean, _inv_std_var = op.SkipLayerNormalization(
+    normalized, _mean, _inv_std_var, skip_sum = op.SkipLayerNormalization(
         input,
         skip,
         gamma,
         beta,
         epsilon=epsilon,
-        _outputs=3,
+        _outputs=4,
+        _domain="com.microsoft",
+    )
+    return normalized, skip_sum
+
+
+# Fusion rule for Add + SkipLayerNormalization
+def _skip_layer_norm_add_bias_pattern(op, input, skip, gamma, beta, bias, epsilon, stash_type):
+    bias_sum = op.Add(input, bias)
```

Comment:
So, I guess there are two things that can be added to input, before normalization? The name "bias" seems to mean something, what is "skip" ... is there any more meaningful name for it?

Reply:
That is correct, two things can be added to the input
- one is the bias (that is added to the input)
- secondly the skip input, coming from the previous layer (layer norm in the case of whisper for example)
- in this case `skip` makes sense, maybe `skip_input` could make sense, however the ort-contrib ops just utilize `skip` so maybe that should work